components/sendgrid/actions/list-global-supressions/list-global-supressions.js

```diff
@@ -0,0 +1,77 @@
+const validate = require("validate.js");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-list-global-supressions",
+  name: "List Global Supressions",
```

Comment:
**-Minor-**
"Suppressions" in the key, name, and folder/filename is missing a "p".

```suggestion
  key: "sendgrid-list-global-suppressions",
  name: "List Global Suppressions",
```